src/js/components/Form/stories/ValidateOnChange.js

```diff
@@ -38,8 +41,8 @@ export const ValidateOnChange = () => (
             ]}
           />
 
-          <FormField label="Email" name="email" required>
-            <TextInput name="email" type="email" />
+          <FormField label="Address" name="address" required>
```

Comment:
Any reason we changed from email to address?

Reply:
If we have email, we were not checking for the email format case from the form validation. It throws an error saying improper format only when we click on submit (as we pass type to TextInput). Ideally, it would be better if we use MaskedInput for email.
So, I preferred a simple use case depicting only what we wanted to demo in the story. 'Address' was something which was on top of my mind at that moment.
Will try to update the story with First Name & Last Name as its more appropriate.